src/Symfony/Component/PropertyAccess/PropertyAccessor.php

```diff
@@ -829,8 +848,11 @@ private function camelize($string)
      *
      * @return array|null An array containing the adder and remover when found, null otherwise
      */
-    private function findAdderAndRemover(\ReflectionClass $reflClass, array $singulars)
+    private function findAdderAndRemover(\ReflectionClass $reflClass, array $singulars, PropertyMetadata $metadata = null)
     {
+        if ($metadata && $metadata->getAdder() && $metadata->getRemover()) {
```

Comment:
I'm just mimicking the actual `PropertyAccess` behavior, in which both methods must be defined to activate this kind of accessors. You're right, docs should warn about this.